packages/contracts/tests/nft.js

```diff
@@ -140,11 +140,29 @@ async function test() {
     })
     const token_owners = await aliceUseContract.nft_tokens_for_owner({account_id: 'alice.test.near'})
 
+    const media = await aliceUseContract.claim_media( { args: {tokenMetadata: {media: ('abc')}},
+        gas: CONTRACT_CLAIM_GAS, amount: CONTRACT_CLAIM_PRICE })
+    
+    // approval tests
+
+    const before_approve = await aliceUseContract.nft_is_approved({token_id: media.id, approved_account_id: 'market.test.near'})
+
+    await aliceUseContract.nft_approve({args: { token_id: media.id, account_id: 'market.test.near' }, gas: CONTRACT_CLAIM_GAS, amount: CONTRACT_CLAIM_PRICE})
```

Comment:
Could you write a couple of logs of how media object looks after `approval`, `revoke` @0x5CE ?